### ISSUE TITLE
missing __version__

### ISSUE THREAD

**Issue by mbnshtck** [gweis/isodate#62]

**Comment by hugovk** [comment]
Fixed in the fork at https://github.com/isodate/isodate/pull/10.

**Comment by hugovk** [comment]
I can remake the PR here if wanted, or you can do a similar thing to get the version of packages easily in your own code:

```python
try:
    # Python 3.8+
    import importlib.metadata as importlib_metadata
except ImportError:
    # <Python 3.7 and lower: python -m pip install importlib-metadata
    import importlib_metadata

print(importlib_metadata.version("isodate"))
```

**Comment by gweis** [comment]
fixed in 0.7
